gluten-flink/docs/Flink.md

```diff
@@ -39,14 +39,16 @@ export PATH=$JAVA_HOME/bin:$PATH
 
 **Get Velox4j**
 
-Gluten for Flink depends on [Velox4j](https://github.com/velox4j/velox4j) commit:7aaa6465cb2c56c8898b737bcc770bae35a94ee4 to call velox. This is an experimental feature.
+Gluten for Flink depends on [Velox4j](https://github.com/velox4j/velox4j) to call velox. This is an experimental feature.
```

Comment:
Can we still mark out a fixed commit? I assume it's in `bigo/velox4j` ?

Reply:
Add in the follow command.